app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):

```

Comment:
Dwarf should be inherited from Player 
Also no need to import ABC in dwarf module

Reply:
Both variants are ok. We can inherit ABC for explicity